packages/ui-components/src/components/Icons.tsx

```diff
@@ -874,6 +877,42 @@ export function initIcons(): void {
                     />
                 </svg>
             ),
+            [UiIcons.FileImage]: (
+                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M13 12C13.5523 12 14 11.5523 14 11C14 10.4477 13.5523 10 13 10C12.4477 10 12 10.4477 12 11C12 11.5523 12.4477 12 13 12Z"
+                        fill={COLORS.default}
+                    />
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M8 8C7.44772 8 7 8.44772 7 9V15C7 15.5523 7.44772 16 8 16H15C15.5523 16 16 15.5523 16 15V9C16 8.44772 15.5523 8 15 8H8ZM15 9H8L8 12.2929L8.93938 11.3536C9.52517 10.7678 10.4749 10.7678 11.0607 11.3536L14.7071 15H15V9ZM8 15V13.7071L9.64649 12.0607C9.84175 11.8654 10.1583 11.8654 10.3536 12.0607L13.2929 15H8Z"
+                        fill={COLORS.default}
+                    />
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M6 14H3V2L9.001 2.002L8.997 5.509C8.997 5.643 9.049 5.77 9.144 5.863C9.237 5.957 9.364 6.01 9.497 6.01L13 6V7H14V5.499C14 5.49199 13.9983 5.48585 13.9965 5.47964C13.995 5.47434 13.9935 5.46899 13.993 5.463V4.533L13.994 4.532L13.993 4.531V4.528H13.99L10.462 1.001H9.387V1.002L2.5 1C2.367 1 2.24 1.053 2.146 1.146C2.053 1.24 2 1.367 2 1.5V14.5C2 14.776 2.224 15 2.5 15H6V14ZM13.048 5L10.001 1.953L9.997 5.009L13.048 5Z"
+                        fill={COLORS.default}
+                    />
+                </svg>
+            ),
+            [UiIcons.FileVector]: (
+                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        fillRule="evenodd"
+                        clipRule="evenodd"
+                        d="M6 14H3V2L9.001 2.002L8.997 5.509C8.997 5.643 9.049 5.77 9.144 5.863C9.237 5.957 9.364 6.01 9.497 6.01L13 6H14V5.499C14 5.49199 13.9983 5.48585 13.9965 5.47964C13.995 5.47434 13.9935 5.46899 13.993 5.463V4.533L13.994 4.532L13.993 4.531V4.528H13.99L10.462 1.001H9.387V1.002L2.5 1C2.367 1 2.24 1.053 2.146 1.146C2.053 1.24 2 1.367 2 1.5V14.5C2 14.776 2.224 15 2.5 15H6V14ZM13.048 5L10.001 1.953L9.997 5.009L13.048 5Z"
+                        fill="#C5C5C5"
```

Comment:
```suggestion
                        fill={COLORS.default}
```
Should it be like this?

Reply:
Yes, my bad, all fills in this case should be `fill={COLORS.default}`